packages/neovim/src/attach/attach.ts

```diff
@@ -2,20 +2,24 @@ import { createConnection } from 'net';
 import * as child from 'child_process';
 
 import { NeovimClient } from '../api/client';
-import { logger } from '../utils/logger';
+import { Logger } from '../utils/logger';
```

Comment:
Would you also make change in `Base.ts`

from

```ts
import { logger as loggerModule } from '../utils/logger';
```
to

```ts
import { Logger } from '../utils/logger';
```

and replace `typeof loggerModule` with `Logger`?